### PR TITLE
MVCA

### PR DESCRIPTION
**Pull request by brookehus**
MVCA clustering algorithm. Paper currently in review.  Needs tests.

 - [x] Implement feature / fix bug
 - [x] Add tests
 - [x] Update changelog

### CODE REVIEW

**Comment by brookehus** [comment]
@msmbuilder/developers this ready for review/merge

File: msmbuilder/lumping/mvca.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+        if n_macrostates is not None:
+            lumper._do_lumping()
+
+        if get_linkage:
+            p = pdist(msm.transmat_, metric=metric)
+            l = scipy.cluster.hierarchy.linkage(p, 'ward')
+
+            lumper.pairwise_dists = p
+            lumper.linkage = l
+            lumper.elbow_data = l[:,2][::-1]
```
Comment (cxhernandez) [inline msmbuilder/lumping/mvca.py]: space after comma

File: msmbuilder/utils/divergence.py [hunk 2]
```diff
@@ -23,6 +23,8 @@ def manual_kl_divergence(P, Q, scalar=True):
                 temp += entry * np.log(entry / Q[row][i])
         vec.append(temp)
     result = np.array(vec)
+    # sometimes there are very tiny negative values close to zero
+    result = [np.max([i,0]) for i in result]
```
Comment (cxhernandez) [inline msmbuilder/utils/divergence.py]: space after comma

File: examples/Coarse-graining-with-MVCA.ipynb [hunk 3]
```diff
@@ -0,0 +1,271 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Minimum Variance Cluster Analysis\n",
+    "We are going to use a minimum variane criterion with the\n",
```
Comment (cxhernandez) [inline examples/Coarse-graining-with-MVCA.ipynb]: typo here

File: examples/Coarse-graining-with-MVCA.ipynb [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from msmbuilder.example_datasets import QuadWell\n",
+    "from msmbuilder.msm import MarkovStateModel\n",
+    "from msmbuilder.lumping import MVCA\n",
+    "import numpy as np\n",
+    "import scipy.cluster.hierarchy\n",
+    "% pylab inline"
```
Comment (cxhernandez) [inline examples/Coarse-graining-with-MVCA.ipynb]: use `%matplotlib inline%, I think `pylab` is deprecated

**Pull request merged**